OrbitGl/DataManager.cpp

```diff
@@ -141,3 +141,17 @@ const std::shared_ptr<Process>& DataManager::selected_process() const {
   CHECK(std::this_thread::get_id() == main_thread_id_);
   return selected_process_;
 }
+
+void DataManager::SelectTracepoint(const TracepointInfo& info) {
+  std::pair<std::string, std::string> pair = std::make_pair(info.category(), info.name());
+  if (!selected_tracepoints_.contains(pair)) selected_tracepoints_.emplace(pair);
```

Comment:
nitpick: you can actually do `emplace(info.category(), info.name());` it should create the pair in place.